python/vineyard/cli.py

```diff
@@ -48,14 +68,13 @@ def vineyard_argument_parser():
 
 
 def exit_with_help():
+    """Utility to exit the program with help message in case of any error!"""
     optparser.print_help(sys.stderr)
     sys.exit(-1)
 
 
-__vineyard_client = None
-
-
 def connect_vineyard(args):
+    """Utility to create a vineyard clinet using an IPC or RPC socket!"""
```

Comment:
`clinet` -> `client`

Reply:
And those doc string should not end with a `!`... It's too aggressive. A period will do.